### DISCUSSION TITLE
NOT FOUNT (404)

### DISCUSSION THREAD

**Discussion by principlebrothers** [vercel/community#4927] [Bug Report]
### Page to Investigate

https://yarysa-p5bcvfjaz-principlebrothers.vercel.app/onboard-hospital

### Steps to Reproduce

When I run the code on my pc/local it works fine. There are no issues with the path. However, deploying it to vercel display the error below.
![Screenshot from 2023-11-22 06-41-02](https://github.com/vercel/community/assets/92553079/ab367f89-e2bf-462c-b280-d4401daad1b0)

I am using React with vite. I checked the issues page and realised someone has reported similar issue but with Ruby. It has not been answered so I couldn't get anything out of it. 
I would like to know the cause of the problem. 

Thank you

Best Regards,
Ernest

**Comment by amyegan** [comment]
Hey Ernest! There are a lot of possible causes for a [404: RESOURCE_NOT_FOUND](https://vercel.com/docs/errors/application-error-codes#404:-resource_not_found) error. If you share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), I can help you investigate 🙏

**Reply by principlebrothers** [reply]
Hello @amyegan, thank you for for the reply. 
I hope the attached minimal reproducible eample helps
`function App() {
  return (
    <>
      <Router>
        <Routes>
          <Route path="/" element={<LandingPage />} />
          <Route path="/onboard-hospital" element={<LoginPage />} />
          <Route path="/blogs" element={<BlogPage />} />
          <Route path="*" element={<Error />} />
        </Routes>
      </Router>
      <ToastContainer />
    </>
  );
}

export default App;` with this code that I am showing above 👆🏾, the onboard-hospiital links to the page currently giving the 404 error. 
The content of the onboard-hospitals page is below `import React, { useRef } from 'react';
import { RiArrowGoBackFill } from 'react-icons/ri';
import emailjs from '@emailjs/browser';
import { toast } from 'react-toastify';
import promoLogo from '../../assets/sign-up-cover.jpeg';
import yarysaLogo from '../../assets/yarysaLogo.svg';
import Union from '../../assets/Union.svg';
import medicalEquipment from '../../assets/medicalEquipment.svg';
import styles from './Login.module.css';
import 'react-toastify/dist/ReactToastify.css';

function Login() {
  const promoForm = useRef();

  const submitPromoForm = (e) => {
    e.preventDefault();
    const serviceID = import.meta.env.VITE_YARYSA_SERVICE_ID;
    const templateID = import.meta.env.VITE_YARYSA_ONBORDING_TEMPLATE_ID;
    const userID = import.meta.env.VITE_YARYSA_USER_ID;

    emailjs.sendForm(serviceID, templateID, promoForm.current, userID).then(
      () => {
        toast.success('Message sent successfully', {
          position: 'top-center',
          autoClose: 5000,
          hideProgressBar: false,
          closeOnClick: true,
          pauseOnHover: true,
          draggable: true,
          progress: undefined,
          theme: 'light',
        });
        e.target.reset();
      },
      () => {
        toast.error('Message not sent', {
          position: 'top-center',
          autoClose: 5000,
          hideProgressBar: false,
          closeOnClick: true,
          pauseOnHover: true,
          draggable: true,
          progress: undefined,
          theme: 'light',
        });
      },
    );
  };
  return (
    <section className={styles.container}>
      <div className={styles.back__btn}>
        <a href="/" aria-label="return">
          <RiArrowGoBackFill />
        </a>
      </div>
      <div className={styles.left__side}>
        <img src={Union} alt="Union" className={styles.union} />
        <div className={styles.sign__in__info}>
          <div className={styles.logo__container}>
            <img src={yarysaLogo} alt="yarysaLogo" />
          </div>
          <img
            src={medicalEquipment}
            alt="medicalEquipment"
            className={styles.equipment}
            width="504px"
            height="511px"
          />
          <h1>Onboard Hospital</h1>
          <p>Experience the Difference with a Free 3-Month Trial</p>
        </div>
        <div className={styles.sign__in__form}>
          <form ref={promoForm} onSubmit={submitPromoForm}>
            <input
              type="text"
              name="hospital_name"
              placeholder="Enter hospital name"
              required
            />
            <input
              type="text"
              name="contact_person_name"
              placeholder="Enter contact fullname "
              required
            />
            <input
              type="email"
              name="contact_person_email"
              placeholder="Enter contact email"
              required
            />
            <input
              type="number"
              name="contact_person_phone"
              placeholder="Enter contact phone number"
              required
            />
            <input type="submit" value="Sign Up" />
          </form>
        </div>
        <div className={styles.allright_reverse}>
          <span>All rights Reserved</span>
          <p>
            &copy;
            {' '}
            <span>Devdex Software</span>
            {' '}
            {new Date().getFullYear()}
          </p>
        </div>
      </div>
      <div className={styles.right__side}>
        <div className={styles.right__side__content}>
          <img src={promoLogo} alt="Promo-flyer" />
        </div>
      </div>
    </section>
  );
}

export default Login;`

**Comment by principlebrothers** [comment]
I hope the information above helps
